FP2-blobs-out.txt

```diff
@@ -0,0 +1,427 @@
+target/product/FP2/system/vendor/lib/libmmipstreamsourcehttp.so
```

Comment:
Yes it is. Anyway I'm going to close this PR and make a new one which I tested from scratch and worked fine.